auth/src/main/java/com/firebase/ui/auth/ui/email/WelcomeBackPasswordPrompt.java

```diff
@@ -141,10 +140,10 @@ public void onDonePressed() {
     }
 
     private void validateAndSignIn() {
-        validateAndSignIn(mEmail, mPasswordField.getText().toString());
+        validateAndSignIn(mPasswordField.getText().toString());
     }
 
-    private void validateAndSignIn(final String email, final String password) {
+    private void validateAndSignIn(String password) {
         // Check for null or empty password
         if (TextUtils.isEmpty(password)) {
             mPasswordLayout.setError(getString(R.string.fui_required_field));
```

Comment:
Should we `return` if the password is empty?

Reply:
We do, it looks like the diff hides it 😉